aexpect/remote.py

```diff
@@ -500,17 +502,18 @@ def _remote_scp(
     :param login_timeout: The maximal time duration (in seconds) to wait for
             each step of the login procedure (i.e. the "Are you sure" prompt or
             the password prompt)
-    :raise SCPAuthenticationError: If authentication fails
-    :raise SCPTransferTimeoutError: If the transfer fails to complete in time
-    :raise SCPTransferFailedError: If the process terminates with a nonzero
+    :param method: The method to use for transfer ("scp" or "rsync")
+    :raise AuthenticationError: If authentication fails
+    :raise TransferTimeoutError: If the transfer fails to complete in time
+    :raise TransferFailedError: If the process terminates with a nonzero
             exit code
     :raise SCPError: If some other error occurs
     """
     password_prompt_count = 0
     timeout = login_timeout
     authentication_done = False
 
-    scp_type = len(password_list)
+    transfer_type = len(password_list)
```

Comment:
Well I wouldn't call it scp_type nor transfer_type, it's just number of passwords user provided. But this was here before so no obligation to change that (I was actually surprised by the whole function, could use some love, but I haven't seen anything obviously dangerous so again, no need to change at this point)